### ISSUE TITLE
Allow distinct entity manager instead of default

### ISSUE THREAD

**Issue by markitosgv** [gesdinet/JWTRefreshTokenBundle#12]
Hi,

If you want to use this bundle with another entity manager instead of default, this fails.

Then you need to override in your **app/config/services.yml** gesdinet.jwtrefreshtoken.refresh_token_manager service, injecting your custom entity_manager like this:

```yaml
gesdinet.jwtrefreshtoken.refresh_token_manager:
  class: Gesdinet\JWTRefreshTokenBundle\Doctrine\RefreshTokenManager
  arguments: [@doctrine.orm.custom_entity_manager, %gesdinet.jwtrefreshtoken.refresh_token.class%]
```

Another example: if you are using FOSUserBundle you maybe want to inject **fos_user.entity_manager** service like this:

```yaml
gesdinet.jwtrefreshtoken.refresh_token_manager:
  class: Gesdinet\JWTRefreshTokenBundle\Doctrine\RefreshTokenManager
  arguments: [@fos_user.entity_manager, %gesdinet.jwtrefreshtoken.refresh_token.class%]
```
To do:

- Allow in bundle config to change entity manager that will be injected to RefreshTokenManager service.

**Comment by pitpit** [comment]
should be implemented in d72dde693ad29fe2d4209114e1b694d51706afc8

**Comment by markitosgv** [comment]
closed by #41
